apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -110,6 +110,13 @@
          * Limit the length of the operationName to prevent errors when inserting elasticsearch
          **/
         public static int OPERATION_NAME_THRESHOLD = 500;
+
+        /*
+        * service properties
+        * e.g.
+        *   agent.properties[org]=apache
+        */
+        public static Map<String, String> PROPERTIES = new HashMap<String, String>();
```

Comment:
The properties should be instance level, according to your latest update. This should be named as `INSTANCE_PROPERTIES`, and documents is required to update.

Reply:
Also, this config should be next to `INSTANCE_UUID`.